source/python/files.rst

```diff
@@ -173,7 +173,7 @@ Exercises
 
 ----
 
-This :download:`tab delimited text file <../data/donor_by_cancer_type.tsv>` is derived from the `Pan-Cancer Analysis of Whole Genomes project <https://dcc.icgc.org/pcawg>`_. Using this file, answer the following using plain python only (no 3rd party libraries).
+This :download:`tab delimited text file <../data/donor_by_cancer_type.tsv>` is derived from the `Pan-Cancer Analysis of Whole Genomes project <https://dcc.icgc.org/pcawg>`_. Using this file, answer the following using plain python only (no 3rd party libraries). Note that we have put a copy of this file in the `data/` directory in your home directory.
 
```

Comment:
I think this is better as an issue than an edit. In part because this page is readable by anyone and in part because it's an opportunity to provide information on how the reader could save this linked file using python in a jupyter notebook.

So I'm going to close this and raise it as an issue. I'll also pose the question to you, do you know how you might use python to read the contents of a file off the internet and save it on the same machine your python process is running?